src/main/java/crazypants/enderio/machine/soul/SoulBinderEnderCystalRecipe.java

```diff
@@ -1,28 +1,29 @@
 package crazypants.enderio.machine.soul;
 
-import static crazypants.enderio.machine.MachineObject.itemMaterial;
 
 import crazypants.enderio.config.Config;
+import crazypants.enderio.init.ModObject;
 import crazypants.enderio.material.material.Material;
 import crazypants.util.CapturedMob;
 import net.minecraft.item.ItemStack;
+import net.minecraft.util.ResourceLocation;
 
 public class SoulBinderEnderCystalRecipe extends AbstractSoulBinderRecipe {
 
   public static SoulBinderEnderCystalRecipe instance = new SoulBinderEnderCystalRecipe();
 
   private SoulBinderEnderCystalRecipe() {
-    super(Config.soulBinderEnderCystalRF,Config.soulBinderEnderCystalLevels, "SoulBinderEnderCystalRecipe", "SpecialMobs.SpecialEnderman", "Enderman");
+    super(Config.soulBinderEnderCystalRF,Config.soulBinderEnderCystalLevels, "SoulBinderEnderCystalRecipe", new ResourceLocation("SpecialMobs.SpecialEnderman"), new ResourceLocation("minecraft:enderman"));
```

Comment:
Do not hardcode "minecraft:" into RLs.

1. *Always* use the 2-param RL constructor
2. In this case, do not use any domain, "minecraft" is the default.